include/zephyr/pm/device.h

```diff
@@ -635,6 +635,17 @@ bool pm_device_is_powered(const struct device *dev);
  */
 int pm_device_driver_init(const struct device *dev, pm_device_action_cb_t action_cb);
 
+/**
+ * @brief Deinit a device driver
+ *
+ * This function must be called at the end of a driver deinit function.
```

Comment:
Shouldn't it be called at the beginning of the de-initialization ? Since if the device is active it will return -EBUSY and the de-initialization should be postponed ?

Reply:
yes, will update description :)

Reply:
done